ioc/Config/Jail/JailConfig.py

```diff
@@ -37,7 +37,7 @@ class JailConfig(ioc.Config.Jail.BaseConfig.BaseConfig):
     legacy: bool = False
     jail: typing.Optional['ioc.Jail.JailGenerator']
     data: dict = {}
-    ignore_host_defaults: bool = False
+    ignore_user_defaults: bool
```

Comment:
what's this mean now?

Reply:
The boolean value was shared across all instances of JailConfig. Initializing it within __init__ makes it an instance property rather than a singleton.